publication-doi-commons/src/test/java/no/unit/nva/publication/doi/PublicationMapperTest.java

```diff
@@ -52,13 +55,13 @@ void fromDynamoStreamRecordNewImage() {
     }
 
     @Test
-    void fromDynamoStreamRecordOldImage() {
-        DynamodbStreamRecordJsonPointers jsonPointers = new DynamodbStreamRecordJsonPointers(
-            DynamodbImageType.OLD);
-        Builder daoBuilder = Builder.createValidPublication(FAKER, jsonPointers, StreamViewType.OLD_IMAGE.toString());
-        var dynamodbStreamRecord = daoBuilder.build().asDynamoDbStreamRecord();
+    void fromDynamoStreamRecordOldImage() throws JsonProcessingException {
```

Comment:
Failing test due to bug in TestData generator.

Reply:
From what I can see the reason why it was failing it due to passing in wrong instance of jsonPointer where it is looking for newImage and not oldImage as the stream view type is set to.